src/Skills.jsx

```diff
@@ -161,6 +161,9 @@ class Skills extends Component {
          },
        });
      })
+     .then(() => {
+       browserHistory.replace('/skills/');
```

Comment:
If you use the same URL then yes, you will get the 404 view, but the point is that there's no reason to keep that URL in the browser history when they can no longer access it. Of course they can attempt to revisit the same URL but they'll just get the 404 message, as they should